appConfig.ts

```diff
@@ -28,6 +28,10 @@ export const commento: boolean = false
 export const commentoUrl: string = 'https://cdn.commento.io'
 //export const commentoUrl: string = 'https://commento.your-blog.com'
 
+// disqus commenting system
+export const disqus: boolean = true
```

Comment:
Could you please switch the default to false?

Reply:
Discussion (I haven't made up my mind): What do you think about diabling disqus when commento is enabled and vice versa, so only one commenting system is allowed at the same time?

In this case we would also need to change the config to something like
```
type CommentingTypes = 'commento' | 'disqus' | false
export const commenting: CommentingTypes = false
...
The `processEnv` logic would need to be updated too. I am leaning towards this change as I don't see that anyone would like to use both systems in parallel.
```

Reply:
Totally agree that it doesn't make sense to have both active at the same time. However I always find it weird when you have a choice, then a specific setting related to that choice, it kinda makes the usage harder IMO. It think it's easier to have a common way to configure all the settings

```
settingsSectionXXXEnabled: TRUE | FALSE
then all the settings related to that section
```

Reply:
otherwise switched default to false